package.json

```diff
@@ -13,7 +13,10 @@
     "express": "^4.17.1",
     "express-handlebars": "^3.1.0",
     "express-session": "^1.16.2",
-    "mysql": "^2.17.1",
-    "pm2": "^3.5.1"
+    "mysql": "^2.16.0",
+    "pm2": "^3.5.1",
+    "forever": "^0.15.3",
```

Comment:
Probably use pm2 instead of forever due to security vulnerabilities in packages included by forever.